src/parser/warlock/demonology/modules/pets/PetTimelineTab/TabComponent/Timeline.scss

```diff
@@ -0,0 +1,60 @@
+.lane {
```

Comment:
Please don't create such a specific selector as it will interfere with other places this selector is used